lib/iex/lib/iex/helpers.ex

```diff
@@ -393,7 +393,14 @@ defmodule IEx.Helpers do
   end
 
   @doc """
-  Prints information about the given data type.
+  Prints information about the data type of a given term.
+  
+  ## Examples
+
+      i([1,2,3])
+      i("Elixir rocks!")
+      i(:some_atom)
+      i(fn x -> x * 2 end)
```

Comment:
If we have output in the examples, I would only have one example (any data type is fine really), and then say something like "this will work for all Elixir terms".